### PR TITLE
Add ywluogg into golang verified catalogs' collaborators and as tektoncd-catalog org member

### PR DESCRIPTION
**Pull request by ywluogg**

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: org/org.yaml [hunk 1]
```diff
@@ -888,7 +889,16 @@ orgs:
         - PuneetPunamiya
         - sm43
         - QuanZhang-William
+        - ywluogg
         privacy: closed
         repos:
           golang: read
+      git-clone.maintainers:
+        description: The maintainers of golang catalog
```
Comment (QuanZhang-William) [inline org/org.yaml:897]: git-clone catalog?

Comment (ywluogg) [reply org/org.yaml]: This is outdated

File: org/org.yaml [hunk 2]
```diff
@@ -866,6 +866,7 @@ orgs:
     - priyawadhwa
     - PuneetPunamiya
     - sm43
+    - ywluogg
```
[the remark this answers is not in the document]
Comment (QuanZhang-William) [reply org/org.yaml:869]: +1. I think you need admin status for bootstrapping purpose.

cc @vinamra28

**Review by vinamra28 — COMMENTED** [review]
/lgtm

**Pull request merged**